local-cli/eject/eject.js

```diff
@@ -84,6 +129,29 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons');
+    const hdpiPath = 'android/app/src/main/res/mipmap-hdpi/ic_launcher.png';
```

Comment:
Does this work on windows? I think this path separator may not work there.

Its not great that the path is duplicated several times here